go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/ffuf/ffuf
+
+go 1.12
```

Comment:
Can we make this `go 1.11` or is it possible to leave it out altogether?

Reply:
Removing wasn't possible as it auto generates, however downgrading it to go 1.11 works fine.